chroma_api/job.py

```diff
@@ -31,7 +31,7 @@ def dehydrate_locked_item_content_type_id(self, bundle):
         if hasattr(locked_item, "content_type"):
             return locked_item.content_type.id
         else:
-            return ContentType.objects.get_for_model(locked_item)
+            return ContentType.objects.get_for_model(locked_item).id
```

Comment:
#2090

Reply:
Interesting we haven't hit this before, this code has not been changed in 8 years :)